README.md

```diff
@@ -315,6 +316,15 @@ trainer.test()
 
 ```
 
+## Computational performances
+
+The data generation process is rather efficient within DeepRank2. As an example, we show the time required to process the tutorials' PDB files available at [this](https://zenodo.org/record/8187806) Zenodo address. Atomic resolution and `distance_cutoff` of 5.5 Å have been used for both PPIs and SRVs, and the SRVs `radius` was set to 10 Å. The experiments are done on Apple M1 Pro, using 1 CPU only. Measures are computed on 100 data points for PPIs, and 96 for SRVs. 
+
+|      | [Features modules](https://deeprank2.readthedocs.io/en/latest/features.html) used                       | Comments                                                                                                                                                                                                                                       |          Data processing speed **[seconds/structure]**        |    Memory **[megabyte/structure]**   |
+|------|---------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------:|:--------------------------------:|
+| PPIs | `components`, `contact`, `exposure`, `irc`, `secondary_structure`, `surfacearea`, 33 features in total. | `conservation` feature module was not used because PSSM files were not available for the data.                                                                                                                                                 | graph only: **3.58** (std 0.27) <br />graph+grid: **12.92** (std 1.39) | graph only: **0.54** (std 0.07) <br />graph+grid: **16.09** (std 0.44) |
+| SRVs | `components`, `contact`, `exposure`, `irc`, `secondary_structure`, `surfacearea`, 26 features in total.                        | Same as above for `conservation`. | graph only: **1.82** (std 0.37)   <br />graph+grid: **2.58** (std 0.66) | graph only: **0.05** (std 0.01)   <br />graph+grid: **8.52** (std 8.50)|
```

Comment:
I would remove the first two columns from the table and instead make a general statement about the features used above (especially because it's now the same for both).